.github/workflows/neard_release.yml

```diff
@@ -62,6 +62,19 @@ jobs:
           fi
           aws s3 cp --acl public-read latest s3://build.nearprotocol.com/nearcore/$(uname)/${BRANCH}/latest
 
+      - name: Trigger packer image creation workflow
+        if: github.event_name != 'workflow_dispatch' && github.event_name == 'release'
+        run: |
+          SHORT_SHA=$(git rev-parse --short HEAD)
+          COMMIT=$(git rev-parse HEAD)
+          TAG=$(git branch -r --contains=${{ github.ref_name }} | head -n1 | cut -c3- | cut -d / -f 2)
+        
+          curl -L -X POST -H "Accept: application/vnd.github+json" \
+          -H "Authorization: Bearer ${{ secrets.PAGODAPLATFORM_GITHUB_TOKEN }}" \
+          -H "X-GitHub-Api-Version: 2022-11-28" \
+          https://api.github.com/repos/PagodaPlatform/pkr-node/dispatches \
+          -d '{"event_type":"packer-build","client_payload":{"image-name":"near-node-${TAG}-${SHORT_SHA}","neard-binary-s3-uri":"s3://build.nearprotocol.com/nearcore/Linux/${TAG}/${COMMIT}/neard"}}'
```

Comment:
branch and tags are often different. Is better to use branch here, as in line 58

Reply:
Can you elaborate?
I am  only running this when a release is made `github.event_name == 'release'` and want to extract the tag name and use that in the GCP image name

Reply:
Sure. For example [this RC release](https://github.com/near/nearcore/releases/tag/1.40.0-rc.1) tag is 1.40.0-rc.1 but the branch is is 1.40.0 ( see logs in https://github.com/near/nearcore/actions/runs/9289993055/job/25565203631).
This means the the image building workflow will fail to find neard binary at s3://build.nearprotocol.com/nearcore/Linux-x86_64/1.40.0-rc.1/ because it was published in s3://build.nearprotocol.com/nearcore/Linux-x86_64/1.40.0/

Reply:
I See. thanks. I'll will update

Reply:
@andrei-near I have updated it.
Using TAG for the GCP image name and BRANCH for the binary URL. LMK if this makes sense.

Reply:
See line 59:
```
# in case of Release triggered run, branch is empty
if [ -z "$BRANCH" ]; then
   BRANCH=$(git branch -r --contains=${{ github.ref_name }} | head -n1 | cut -c3- | cut -d / -f 2)
fi
```

This is because fetch depth is 0 in case of releases (line 45).
What you define here as TAG, is actually the BRANCH and is defined using same command as above for BRANCH. Sorry, I may have missed to read the var assignment code in previous iterations. 

I'd just copy the branch definition for above step

Reply:
@andrei-near  Updated